contracts/stellar-axelar-example/src/contract.rs

```diff
@@ -18,7 +16,7 @@ use crate::storage;
 
 #[contract]
 #[derive(InterchainTokenExecutable, AxelarExecutable)]
-pub struct Example;
+pub struct AxelarExample;
```

Comment:
why is this being changed again? thought the other PR already fixed it
https://github.com/axelarnetwork/axelar-cgp-stellar/pull/270/files#diff-185c8e9bb12fa807cb4f17f3e017fd0ef9bc3f625aa36e06030fdd5a5dc6bfc3R19